packages/rpc-augment/src/augment/jsonrpc.ts

```diff
@@ -145,7 +145,7 @@ declare module '@polkadot/rpc-core/types/jsonrpc' {
        * @deprecated Use the runtime interface `api.call.contractsApi.call` instead
        * Executes a call to a contract
        **/
-      call: AugmentedRpc<(callRequest: ContractCallRequest | { origin?: any; dest?: any; value?: any; gasLimit?: any; storageDepositLimit?: any; inputData?: any } | string | Uint8Array, at?: BlockHash | string | Uint8Array) => Observable<ContractExecResult>>;
+      call: AugmentedRpc<(callRequest: ContractCallRequestV1 | { origin?: any; dest?: any; value?: any; gasLimit?: any; storageDepositLimit?: any; inputData?: any } | string | Uint8Array, at?: BlockHash | string | Uint8Array) => Observable<ContractExecResult>>;
```

Comment:
If you run `yarn build:metadata` now, it should actually pull in the latest V2 stuff (assuming there is a version: 2 definition available in contracts/runtime.ts)

Reply:
I ran `yarn build:metadata` seems no file changes

Reply:
Ensure you have master merged in - just did a quick (5 mins, literally), `ContractsApi/2` in here and it generates the new interfaces - https://github.com/polkadot-js/api/pull/5330

(No idea if it is fully correct in regards to `ContractsApi` definitions)

Reply:
I rebased master and it has the new metadata commit...
I see you open a new PR for Contracts V2, do you wanna take it?

Reply:
Happy if you take it - you have done the hard work. But either way ok. (Just wanted to show that the runtime apis _does_ get updated, once defined, on master, in the above PR)

Reply:
I believe your issues (reading below), is that your u64 -> Option<Weight> changes are on the v1 APIs, which are not being generated. My PR adds for v2 (and keeps v1 as-is, with some small historic-types-now updates), hence being generated.

Reply:
I miss the head of the file `yarn polkadot-types-from-chain`, so I should build a latest Substrate node, and run `yarn polkadot-types-from-chain --endpoint ws://127.0.0.1:9944 --output packages/rpc-augment/src/augment/jsonrpc.ts`

Reply:
We don include the latest [static metadata](https://github.com/polkadot-js/api/blob/master/packages/types-support/src/metadata/v14/substrate-hex.ts), so `yarn build:metadata` will pick it up. (Well, "latest" as of this morning - it gets updated once weekly)

Reply:
I port your PR's changes, and rerun `yarn build:metadata`, Doe it look correct now?